src/app/newsFeedPage/page.tsx

```diff
@@ -1,19 +1,65 @@
 'use client';
 
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { IoIosArrowRoundBack } from 'react-icons/io';
 import NavBar from '../../components/userComponents/navBar/navBar';
+import { NewsRow } from '../../types/types';
+import { fetchAllNewsByDate } from '../../supabase/news/queries';
+import NewsDisplay from '../../components/userComponents/NewsDisplay/NewsDisplay';
 
 /**
```

Comment:
you guys redefine this function goBack and button component BackButton multiple times. looks like it might work well to be its own component